File: pset6/mario/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n < 9 and n > 0:
        break

for i in range(1,n+1):
    for c in range(n-i):
        print(" ", end = "")
    
    for a in range(i):
        print("#", end = "")
    print("  ", end = "")    
    
    
    for a in range(i):
        print("#", end = "")
        
    print("")

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while(n<1 || n>8);
    for(int i=0;i<n;i++)
    {  
      for(int k=n-i;k>1;k=k-1)
      {
        printf(" ");
      }
        
      for(int j=0;j<=i;j++)
      {
     
          printf("#");
      }
      printf("  ");
      for(int m=0;m<=i;m++)
      {
     
          printf("#");
      }
      printf("\n");
    }
}